polaris-react/src/styles/shared/_buttons.scss

```diff
@@ -10,7 +10,7 @@
   );
 
   @include recolor-icon(var(--p-icon));
-  @include focus-ring($border-width: var(--p-border-width-1));
+  @include focus-ring($border-width: 1px);
```

Comment:
Again this should probably be consolidated for now. It's a 1px line around an element on focus. I understand shadows will replace this but I think for now we should use the token.